doc/terms_x-road_docs.md

```diff
@@ -38,231 +39,176 @@ This document is licensed under the Creative Commons Attribution-ShareAlike 3.0
 
 ### 6.5 X-Road protocols
 
-**Message Transport Protocol** \[PR-MESSTRANSP\] (*X-tee sõnumitranspordi protokoll*, lühidalt *transpordiprotokoll*) – communications protocol that is used by service client's and service provider's security servers to exchange messages with each other.
-
-**Protocol for Downloading Configuration** \[PR-GCONF\] (*X-tee konfiguratsiooni allalaadimise protokoll*, lühidalt *konfiguratsiooniprotokoll*) – protocol that is used to distribute configuration to security servers of an X-Road instance.
+**Message Transport Protocol** – communications protocol that is used by service client's and service provider's security servers to exchange messages with each other.
 
-**Federation Protocol** \[PR-FGONF\], currently same as\[PR-GCONF\] (*X-teede ühendamise protokoll*) – protocol that is used to distribute configuration between two federated X-Road instances.
+**Protocol for Downloading Configuration** – protocol that is used to distribute configuration to security servers of an X-Road instance.
 
-**Message protocol** \[PR-MESS\] (*X-tee sõnumiprotokoll*, lühidalt *sõnumiprotokoll*) – protocol that is used between information systems and security servers in the X-Road system.
+**Federation Protocol** – protocol that is used to distribute configuration between two federated X-Road instances.
 
-> **X-tee sõnumiprotokoll** on X-tee baasprotokollistiku osa, mis tagab turvaserveri võime töödelda sõnumeid.
+**Message protocol** – protocol that is used between information systems and security servers in the X-Road system.
 
-**Service Metadata Protocol** \[PR-META\] (*X-tee teenuste metaandmete protokoll*, lühidalt *metaandmete protokoll*) – protocol that describes methods that can be used by X-Road participants to discover what services are available to them and download the WSDL files describing these services.
+**Service Metadata Protocol** – protocol that describes methods that can be used by X-Road participants to discover what services are available to them and download the WSDL files describing these services.
 
 ### 6.6 Logging and security
 
-Secure log?
+**Message log** – a log, where exchanged messages are logged and provided with batch signature.
```

Comment:
Change the text as follows (i.e. add _X-road_ word): 
a log, where exchanged X-Road messages are logged and provided with batch signature.